route-rs-runtime/src/processor/mod.rs

```diff
@@ -10,6 +19,7 @@ pub use self::drop::*;
 mod dec_ip_hop;
 pub use self::dec_ip_hop::*;
 
+/// In order for a struct to become a Processor, it must implement this Trait.
```

Comment:
We can remove this comment. It doesn't seem to add any extra info for a user. e.g.
> In order for a struct to become Eq, it must implement Eq.

Reply:
lol that is brutally true